webapp/packages/core-connections/src/ConnectionInfoOriginDetailsResource.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { runInAction, toJS } from 'mobx';
+
+import { injectable } from '@cloudbeaver/core-di';
+import { ProjectsService } from '@cloudbeaver/core-projects';
+import { CachedMapResource, isResourceAlias, ResourceKey, resourceKeyList, ResourceKeyUtils } from '@cloudbeaver/core-resource';
+import { DatabaseConnectionOriginDetailsFragment, GraphQLService } from '@cloudbeaver/core-sdk';
+import { schemaValidationError } from '@cloudbeaver/core-utils';
+
+import { CONNECTION_INFO_PARAM_SCHEMA, IConnectionInfoParams } from './CONNECTION_INFO_PARAM_SCHEMA';
+import {
+  ConnectionInfoActiveProjectKey,
+  ConnectionInfoProjectKey,
+  ConnectionInfoResource,
+  createConnectionParam,
+  isConnectionInfoParamEqual,
+} from './ConnectionInfoResource';
+
+@injectable()
+export class ConnectionInfoOriginDetailsResource extends CachedMapResource<IConnectionInfoParams, DatabaseConnectionOriginDetailsFragment> {
```

Comment:
same here, please use `OriginDetailsFragment` and reexport it with proper name

Reply:
done